src/ibutton.toit

```diff
@@ -59,13 +62,56 @@ The iButton is a small, waterproof, and shock-resistant device that can be used
   can be read using the 1-wire bus.
 */
 class IButton:
-  id/int
+  static COMMAND_WRITE_SERIAL_ ::= 0xD5
+
+  id_/int := ?
   bus_/one_wire.Bus
 
   /**
   Constructs an iButton instance.
   */
-  constructor .bus_ .id:
+  constructor .bus_ .id_:
+
+  id -> int: return id_
 
   stringify -> string:
-    return "IButton-$(%016x id)"
+    return "IButton-$(%016x id_)"
+
+  /**
+  Writes the given $id to the iButton.
+
+  The device must have a changeable ID. A typical device would be the RW1990.
+
+  Depending on the version, the ID bits must be written in inverse order.
```

Comment:
The version of what?

Reply:
Of the RW1990. I updated the doc.
done.